### ISSUE TITLE
Fine tuning model for image editing task

### ISSUE THREAD

**Issue by hardik-uppal** [meituan-longcat/LongCat-Image#41]
What dataset size do you recommend for fine-tuning on a specific image editing task? Would you suggest starting with the mid-training Dev version of the model or the image-edit finetuned one?

**Comment by zqlsnr** [comment]
@hujie-frank Hello, when training edit-LoRA, an error occurs saying "probabilities contain NaN". The same dataset works fine with Flux-Kontext though. Could you please tell me any solutions to this problem?
12/17/2025 06:15:59 - INFO - __main__ - Step=1, Epoch=8, global_step=80, time_all:1.441, time_data:0.063, lr:1.000e-04, s:(ch:16,h:104,w:152), loss:0.0648, aspect_ratio:0.7422, grad_norm:0.0053
get error probabilities contain NaN.
You're using a Qwen2TokenizerFast tokenizer. Please note that with a fast tokenizer, using the `__call__` method is faster than using a method to encode the text followed by a call to the `pad` method to get a padded encoding.
12/17/2025 06:16:28 - INFO - __main__ - Step=1, Epoch=9, global_step=90, time_all:1.432, time_data:0.035, lr:1.000e-04, s:(ch:16,h:104,w:152), loss:0.0938, aspect_ratio:0.7422, grad_norm:0.0047
get error probabilities contain NaN.
You're using a Qwen2TokenizerFast tokenizer. Please note that with a fast tokenizer, using the `__call__` method is faster than using a method to encode the text followed by a call to the `pad` method to get a padded encoding.
12/17/2025 06:16:56 - INFO - __main__ - Step=1, Epoch=10, global_step=100, time_all:1.419, time_data:0.040, lr:1.000e-04, s:(ch:16,h:104,w:152), loss:0.0757, aspect_ratio:0.7422, grad_norm:0.0049
get error probabilities contain NaN.
You're using a Qwen2TokenizerFast tokenizer. Please note that with a fast tokenizer, using the `__call__` method is faster than using a method to encode the text followed by a call to the `pad` method to get a padded encoding.
12/17/2025 06:17:25 - INFO - __main__ - Step=1, Epoch=11, global_step=110, time_all:1.431, time_data:0.042, lr:1.000e-04, s:(ch:16,h:104,w:152), loss:0.0798, aspect_ratio:0.7422, grad_norm:0.0051

**Comment by yayonguan** [comment]
@zqlsnr This situation is most likely related to hyperparameters such as the learning rate. According to the logs, the current learning rate is 1e-4. You can try lowering the learning rate or adjusting the LoRA rank value, which should lead to some improvement.
